devnet-sdk/contracts/bindings/eventlogger.go

```diff
@@ -0,0 +1,62 @@
+package bindings
+
+import (
+	"errors"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+)
+
+// EventloggerMetaData contains all meta data concerning the Eventlogger contract.
+var EventloggerMetaData = &bind.MetaData{
```

Comment:
Can you add the command that generated these bindings to a new justfile?

Reply:
Since this bindings were generated and edited, added the comment to address how this file was made at https://github.com/ethereum-optimism/optimism/pull/14982/commits/8a97bd7209366dbaec222a8c512fdbea933a76cb